src/createServer.js

```diff
@@ -1,3 +1,59 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(
+      req.url,
+      `http://${req.headers.host}`,
+    );
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const words = normalizedURL.pathname.slice(1);
+    const newCase = normalizedURL.searchParams.get('toCase');
+    const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+    const errors = [];
+
+    if (!words) {
+      errors.push({ message: 'Text to convert is required. Correct '
+        + 'request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    }
+
+    if (!newCase) {
+      errors.push({ message: '"toCase" query param is required. Correct '
+        + 'request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
+    }
+
+    if (!(caseNames.includes(newCase)) && newCase?.length > 1) {
+      errors.push({ message: 'This case is not supported. Available cases: '
+          + 'SNAKE, KEBAB, CAMEL, PASCAL, UPPER.' });
```

Comment:
It will be easier to handle if you add/remove one
```suggestion
          + caseNames.join(', ')});
```